### PR TITLE
Remove support of EmbeddedPostgres RMB-789

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: doc/upgrading.md [hunk 1]
```diff
@@ -21,10 +21,34 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
...
+before any calls to PostgresClient or the Verticle. It is also
+possible to provide your own by implemeting the `PostgresTester` interface.
+
+Stopping the Postgres for testing is still done with
+`PostgresClient.stopEmbeddedPostgres` (although the name is kind of
+misleading).
+
+The following command-line options are no longer supported:
+`db_connection=` and `embedpostgres_true`.
+
```
Comment (julianladisch) [inline doc/upgrading.md:46]:
```suggestion

Remove `<groupId>ru.yandex.qatools.embed</groupId>`
`<artifactId>postgresql-embedded</artifactId>` from pom.xml.
```

**Review by hjiebsco — COMMENTED** [review]
Since it is a breaking change, why not cleaning up and/or renaming all legacy methods that contain `Embedded` to be accurate?

File: README.md [hunk 2]
```diff
@@ -259,21 +259,6 @@ we will get your local development server running and populated with test data.
...
-  - for example Postgres: `{"host":"localhost", "port":5432, "maxPoolSize":50,
-    "username":"postgres","password":"mysecretpassword", "database":"postgres",
-    "charset":"windows-1252", "queryTimeout" : 10000}`
-
-  - path defaults to /postgres-conf.json
-
-  - tries to read a file at the path if the path is absolute
-
-  - if file not found or path is relative tries to read a class path resource with that path
-
```
[the remark this answers is not in the document]
Comment (adamdickmeiss) [reply README.md:276]: I'll revert that. I misread the db_connection= code and assumed it was using embedded postgres. It isn't. I'll fix that.

File: domain-models-runtime/src/test/java/org/folio/DemoRamlRestTest.java [hunk 3]
```diff
@@ -82,6 +83,7 @@ public static void setUp(TestContext context) throws IOException {
     Locale.setDefault(Locale.US); // only needed for embedded postgres
 
     // do not use PostgresClient.setPostgresTester here so we check that PostgresTesterEmbedded is working
+    PostgresClient.setPostgresTester(new PostgresTesterContainer());
```
[the remark this answers is not in the document]
Comment (adamdickmeiss) [reply domain-models-runtime/src/test/java/org/folio/DemoRamlRestTest.java]: Yep

**Comment by adamdickmeiss** [comment]
> Since it is a breaking change, why not cleaning up and/or renaming all legacy methods that contain `Embedded` to be accurate?

Suggestions for new names for `startEmbeddedPostgres`  and `stopEmbeddedPostgres` ?

**Comment by hjiebsco** [comment]
> > Since it is a breaking change, why not cleaning up and/or renaming all legacy methods that contain `Embedded` to be accurate?
> 
> Suggestions for new names for `startEmbeddedPostgres` and `stopEmbeddedPostgres` ?

PostgresTester?

**Comment by adamdickmeiss** [comment]
> > > Since it is a breaking change, why not cleaning up and/or renaming all legacy methods that contain `Embedded` to be accurate?
> > 
> > 
> > Suggestions for new names for `startEmbeddedPostgres` and `stopEmbeddedPostgres` ?
> 
> PostgresTester?

OK. Pushed now.

**Review by hjiebsco — APPROVED** [review]
LGTM

**Pull request merged**